### PR TITLE
tests: use crun package

### PR DESCRIPTION
**Pull request by giuseppe**
Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM
@cevich PTAL

This might require new images, no?

**Comment by rhatdan** [comment]
LGTM, Love turning on SELinux...

**Comment by cevich** [comment]
looking...

File: contrib/cirrus/integration_test.sh [hunk 1]
```diff
@@ -37,11 +37,7 @@ case "$SPECIALMODE" in
                 -o CheckHostIP=no $GOSRC/$SCRIPT_BASE/rootless_test.sh ${TESTSUITE}
         ;;
     cgroupv2)
-        # FIXME: use the package once all the fixes are in a release
-        # yum install -y crun
-        setenforce 0
-        yum builddep -y crun
-        (git clone --depth=1 https://github.com/containers/crun && cd crun && ./autogen.sh && ./configure --prefix=/usr && make -j4 && make install)
+        yum install -y crun
```
Comment (cevich) [inline contrib/cirrus/integration_test.sh]: Should be `dnf` :smiley:

Comment (cevich) [reply contrib/cirrus/integration_test.sh]: errr...actually, I think we've already got crun installed in the VM image, no?   Maybe this should be a 'dnf update -y crun'?

Comment (cevich) [reply contrib/cirrus/integration_test.sh]: oh nope, that's the kernel stuff. nvm, 'install' is correct.

**Comment by cevich** [comment]
...Okay yep, just `yum` -> `dnf` then LGTM :smiley:

**Comment by giuseppe** [comment]
> Love turning on SELinux...

looks like it still needs to be turned off for now :/  the tests were failing because of SELinux

**Comment by rhatdan** [comment]
@giuseppe Do you know why?
/lgtm

**Comment by giuseppe** [comment]
> @giuseppe Do you know why?

I'll check it out, one thing I've noticed is that `crun` has the wrong label on Fedora 30 at least:

```
$ ls -Z /usr/bin/crun /usr/bin/runc
                   system_u:object_r:bin_t:s0 /usr/bin/crun
system_u:object_r:container_runtime_exec_t:s0 /usr/bin/runc
```
/hold cancel

**Comment by giuseppe** [comment]
I've opened a PR for container-selinux: https://github.com/containers/container-selinux/pull/76

**Comment by rhatdan** [comment]
Nice catch.

**Comment by rhatdan** [comment]
https://bodhi.fedoraproject.org/updates/FEDORA-2019-d8e99d8528
Please update karma.

**Comment by cevich** [comment]
FYI - if that gets into an updates-testing repo, then rolling new VM images in this repo will scoop it up for use.  Ping me if you need help with that.

**Pull request merged**